### PR TITLE
added thumbnail for Samsung

### PR DESCRIPTION
**Pull request by ivanlewin**

### CODE REVIEW

**Comment by ziegenberg** [comment]
There is already a PR for a Samsung thumbnail. I'll have a look at it and compare them.

**Comment by ivanlewin** [comment]
Thanks, I missed it. I added a black wordmark, which seems to be the current logo, taken from samsung.com

**Comment by ziegenberg** [comment]
This are two Screenshots from this Samsung-thumbnail and the one I would have used (see commit b788ae5fc7632402ae2d4527f7ce6a2bbe06114d in #573). I prefer the black wordmark from this PR, although it appears rather small, therefore might be hard to read on small screens or low resolutions and it's not very square.

Screenshot_1597345652

Screenshot_1597345776

File: build.gradle [hunk 1]
```diff
@@ -6,7 +6,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.0.0'
+        classpath 'com.android.tools.build:gradle:4.0.1'
```
Comment (ziegenberg) [inline build.gradle:9]: Please do not update build tools when committing a thumbnail. This should go into a separate PR.

**Review by ziegenberg — CHANGES_REQUESTED** [review]
There's only one small thing and then it's good to go.

**Comment by ivanlewin** [comment]
Yes, I know it's not perfectly square, it bothers me too. I took the vector from Samsung's website so I think we have to deal with it sadly

**Review by ziegenberg — APPROVED** [review]

**Comment by ziegenberg** [comment]
Fixed #562.

**Pull request merged**